package.json

```diff
@@ -33,50 +33,30 @@
     ]
   },
   "engines": {
-    "node": ">=6.0"
+    "node": ">=8.0"
```

Comment:
Update to latest LTS.

Reply:
Also need to update `.travis.yml` to change the minimum node version for testing.